src/main/java/junit/extensions/ActiveTestSuite.java

```diff
@@ -43,7 +43,6 @@ public void runTest(final Test test, final TestResult result) {
             public void run() {
                 try {
                     // inlined due to limitation in VA/Java
-                    //ActiveTestSuite.super.runTest(test, result);
```

Comment:
I think we should add this comment back, since i clarifies what was inlined.

Reply:
Fixed in 460e488